src/base/ToCpp.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef BASE_TOCPP_H
+#define BASE_TOCPP_H
+
+#ifdef __cplusplus
```

Comment:
I cannot think of a way to justify adding this condition, especially since src/base/ headers are meant to be used by C++ files only. Please remove if possible.

Reply:
Removed